src/main/kotlin/apply/domain/mission/JudgeItem.kt

```diff
@@ -0,0 +1,24 @@
+package apply.domain.mission
+
+import support.domain.BaseEntity
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+
+@Entity
+class JudgeItem(
+    @Column
```

Comment:
r: nullable 여부도 추가해보면 어떨까요?

```suggestion
    @Column(nullable = false)
```